### ISSUE TITLE
Slash command doesn't always work?

### ISSUE THREAD

**Issue by vorburger** [vorburger/minecraft-storeys-maker#40]
While testing #35, I've noticed that the Slash command (not Command Hat!) doesn't always work...

    Command /night
    /time set night
    Title: It's night...

and 

    Command /day
    /time set day
    Title: It's day...

Used to work, and then stopped working... as in /day & night would still show title, server log still show below - but no "effect", day/night did not change.... hm. I'll add some logging to CommandAction to see if we can better understand it (FYI @edewit).

```
[20:09:15 INFO] [ch.vorburger.minecraft.storeys.web.MinecraftVerticle]: Sent to EventBus: {"event":"newCmdday"}
--
  | [20:09:15 INFO] [ch.vorburger.minecraft.storeys.web.ActionsConsumer]: Handling message received on EventBus: {
  | "action" : "command",
  | "command" : "time set day",
  | "code" :
```

I doubt this is multi-user (#35) related, but seems to be some ... Sponge / MC issue?

**Comment by vorburger** [comment]
https://docs.spongepowered.org/stable/en/plugin/scheduler.html#compatibility-with-other-libraries !!!

**Comment by vorburger** [comment]
@edewit you may be interested in having a peek at  5a3d3d6 which fixes this (locally verified & confirmed).

**Comment by edewit** [comment]
@vorburger could this be due to your `/gamerule doDaylightCycle false`

**Comment by vorburger** [comment]
@edewit any command which modifies game state needs to run on Sponge's main thread.

I've fixed it in 5a3d3d6.
